src/5e-SRD-Traits.json

```diff
@@ -95,6 +95,28 @@
     ],
     "url": "/api/traits/dwarven-combat-training"
   },
+  {
+    "index": "tool-proficiency",
+    "races": [
+      {
+        "index": "dwarf",
+        "name": "Dwarf",
+        "url": "/api/races/dwarf"
+      }
+    ],
+    "subraces": [
+      {
+        "index": "hill-dwarf",
+        "name": "Hill Dwarf",
+        "url": "/api/subraces/hill-dwarf"
+      }
+    ],
```

Comment:
If this is given with dwarf, we don't really need to add it with Hill Dwarf. Unless you're granted it a second time with Hill Dwarf.

Reply:
```suggestion
    "subraces": [],
```